SkeletalRepresentationInitializer/Logic/vtkSlicerSkeletalRepresentationInitializerLogic.cxx

```diff
@@ -1699,6 +1698,7 @@ int vtkSlicerSkeletalRepresentationInitializerLogic::DisplayResultSrep()
     AddModelNodeToScene(crestSpoke_poly, "crest spokes for initial object", true, 1, 0, 0);
     AddModelNodeToScene(meshPoly, "skeletal mesh for initial object", true, 0, 0, 0);
     AddModelNodeToScene(curvePoly, "fold curve for initial object", true, 1, 1, 0);
+    return 0;
```

Comment:
If the return is always 0, make the function void instead of int and remove the return.